explainer.html

```diff
@@ -32,13 +32,36 @@
   }],
   github: "w3c/rch-wg-charter",
   localBiblio: {
-    "aidan-2017" : {
+    "carroll-2003" : {
+      title: "Signing RDF Graphs",
+      authors: [
+        "Jeremy J. Carroll"
+      ],
+      href : "https://link.springer.com/chapter/10.1007/978-3-540-39718-2_24",
+      publisher: "International Semantic Web Conference — ISWC 2003, Springer Verlag",
+      pages: "369-384",
+      date : 2003
+    },
+    "kasten-et-al-2014": {
+      title: "Canonical Forms for Isomorphic and Equivalent RDF Graphs: Algorithms for Leaning and Labelling Blank Nodes",
+      authors: [
+        "Andreas Kasten",
+        "Ansgar Scherp",
+        "Peter Schauß"
+      ],
+      href : "https://link.springer.com/chapter/10.1007%2F978-3-319-07443-6_11",
+      publisher: "European Semantic Web Conference — ESWC 2014, Springer Verlag",
+      pages: "146-160",
+      date : 2014
+    },
+    "hogan-2017" : {
       title: "Canonical Forms for Isomorphic and Equivalent RDF Graphs: Algorithms for Leaning and Labelling Blank Nodes",
       authors: [
         "Aidan Hogan"
       ],
       href : "http://aidanhogan.com/docs/rdf-canonicalisation.pdf",
-      publisher: "ACM Transactions on the Web",
+      publisher: "ACM Transactions on the Web, vol. 11, no. 4",
```

Comment:
It looks like [Specref](https://www.specref.org/) is really more for spec refs than full bibliographic references.  Seems like they could use a `publication` property in addition to `publisher`.  Is this best practice considering the limiations?